src/screen/AddUserScreen.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { Button, Input } from 'react-native-elements';
+import { useTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import WebSocket from '../api/socket';
+import Block from '../components/Block';
+import { Navigation } from '../constants/Navigation';
+
+const AddUserScreen = ({
+  navigation,
+  roomName,
+}) => {
+  const [name, setUserName] = useState();
```

Comment:
`const [name, setUserName] = useState();`
I think if you use `setUserName` then the variable should be called `userName`